src/components/optimizationSetup/ElectricalSetupWindow.tsx

```diff
@@ -0,0 +1,177 @@
+import React from "react";
+import { useState } from "react";
+import "normalize.css";
+import "@blueprintjs/core/lib/css/blueprint.css";
+import "@blueprintjs/icons/lib/css/blueprint-icons.css";
+import "./ElectricalSetupWindow.css";
+import {
+    Button,
+    FocusStyleManager,
+    NumericInput,
+    Tag,
+    Divider,
+    FileInput
+  } from "@blueprintjs/core";
+
+FocusStyleManager.onlyShowFocusOnTabs();
+
+const ElectricalSetupWindow: React.FC = () => {
+    const [traceOrientationFile, setTraceOrientationFile] = useState("Choose file...");
+    const [parasiticModelFile, setParasiticModelFile] = useState("Choose file...");
+    const [traceOrientationFileSelected, setTraceOrientationFileSelected] = useState(false);
+    const [parasiticModelFileSelected, setParasiticModelFileSelected] = useState(false);
+
+    const [tableElements, setTableElements] = useState([]);
+    const [numTableElements, setNumTableElements] = useState(0); // These are currently unused
+
+    const handleAddDevice = () => {
+        
+    };
+    const handleRemoveDevice = () => {
+        
+    };
+
+    const handleTraceOrientationFilePath = (event: any) => {
+        setTraceOrientationFile(event.target.files[0].name);
+        setTraceOrientationFileSelected(true);
+    };
+
+    const handleParasiticModelFilePath = (event: any) => {
+        setParasiticModelFile(event.target.files[0].name);
+        setParasiticModelFileSelected(true);
+    };
+    
+    return (
+        <div className="mainDiv">
+            <div className="electricalSetupText">Electrical Setup</div>
```

Comment:
The title here is not needed, since it is already in the popup that this component is being called in.